_alerts/2016-11-01-alert-retirement.md

```diff
@@ -49,15 +50,15 @@ receive the final alert.
 |---|---|---|
 |Pre-final Alert Posts|Posts on Bitcoin.org, various forums, and various mailing lists that the Alert system will be retired|2016-11-01|
 |Pre-final Alert|The alert itself warning that the Alert system will be retired|2016-11-02|
-|Final Alert|Max sequence Alert to disable the Alert system|2017 (Will coincide with Bitcoin Core 0.14 Release Candidate process)|
+|Final Alert|Max sequence Alert to disable the Alert system|2017-01-19|
 |Alert key released|The Alert key will be made publicly available|1-2 months after the Final Alert|
```

Comment:
According to @gmaxwell, there are some vulnerabilities in the Alert handling code that could be exploited if the alert key were released. He said that he would wait to publish it until after people have stopped using vulnerable software.